package.json

```diff
@@ -48,14 +49,14 @@
     "ejs": "^2.5.1",
     "enzyme": "^3.5.0",
     "enzyme-adapter-react-15": "^1.1.0",
-    "eslint": "^3.17.1",
-    "eslint-config-airbnb": "^9.0.1",
-    "eslint-plugin-import": "^1.12.0",
-    "eslint-plugin-jest": "^20.0.3",
-    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint": "^5.10.0",
```

Comment:
Dependency updates are not mentioned in the commit message.

Reply:
Thanks @KKoukiou. The eslint update is due to `eslint-plugin-jsx-a11y` depends on it. And due to this dependence, `eslint-plugin-jsx-a11y` does not work for a long time. So I fix it in this PR.

Reply:
Ah right :) thanks!

Reply:
@KKoukiou, happy holidays!